aibolit/ast_framework/java_class_decomposition.py

```diff
@@ -60,6 +100,8 @@ def decompose_java_class(class_ast: AST, strength: str) -> List[AST]:
             for node in component
             if usage_graph.nodes[node]["type"] == "method"
         }
+        if ignore_getters:
```

Comment:
Do I understand you correctly. Setters and getters are used in component creation, but filtered out from each component? Should we filter them out before graph is splitted on components?

Reply:
as discussed we leave as it is